packages/@aws-cdk/aws-sns/lib/topic-ref.ts

```diff
@@ -41,6 +41,11 @@ export abstract class TopicRef extends cdk.Construct implements events.IEventRul
    */
   private eventRuleTargetPolicyAdded = false;
 
+  /**
+   * The number of sourcs that were registered, used as the Sid for policy statements.
```

Comment:
type "sourcs"

Reply:
How do I make so many typos? What is happening?